accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -439,16 +430,6 @@ Windows-specific:
 * **Thread**
     - `SetApartmentState(ApartmentState)`
 
-### System.Xaml.Permissions
-
-* **XamlAccessLevel**
-    - `AssemblyAccessTo(Assembly)`
-    - `AssemblyAccessTo(AssemblyName)`
-    - `get_AssemblyAccessToAssemblyName()`
-    - `get_PrivateAccessToTypeName()`
-    - `PrivateAccessTo(String)`
-    - `PrivateAccessTo(Type)`
-
```

Comment:
it's defined in `System.Windows.Extensions.dll` which is marked as Windows-specific

https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Windows.Extensions/ref/System.Windows.Extensions.cs#L135-L147